pwiz_tools/Skyline/TestFunctional/LibraryBuildTest.cs

```diff
@@ -110,7 +110,7 @@ private void MainTest()
                 Assert.AreEqual(z, key.Charge);
                 Assert.AreEqual(adduct, key.Adduct);
                 Assert.AreEqual(caffeineInChiKey, key.Target.ToString());
-                var viewLibPepInfo = new ViewLibraryPepInfo(key);
+                var viewLibPepInfo = new ViewLibraryPepInfo(key, null);

```

Comment:
Feels like this could have a default null value to remove the need for this explicit "null" use.

Reply:
Agreed, thanks.